crates/nu-cli/src/commands/ls.rs

```diff
@@ -44,12 +42,6 @@ impl WholeStreamCommand for Ls {
                 "Only print the file names and not the path",
                 Some('s'),
             )
-            .switch(
-                // Delete this
```

Comment:
Funny, this comment somehow snuck into this from a previous commit of mine (totally unintentionally).